src/ActionBar/_ActionBarActions.test.js

```diff
@@ -0,0 +1,14 @@
+import ActionBar from './ActionBar';
+import { mount } from 'enzyme';
+import React from 'react';
+
+describe('ActionBarActions Prop spreading', () => {
```

Comment:
Should just reference `ActionBarActions` (the prop spreading is covered by the actual `test`).